src/core/conf.ml

```diff
@@ -82,3 +87,26 @@ let get_file_regex json =
 (* Get content corresponding to [template] field *)
 let get_user_template json = 
   member json ["template"] 
+
+(* Update the old header field *)
+let update_auto header =
+  Writer.write auto_file header
+  |> ignore
+
+(* Get the old header *)
+let get_old_header value =
```

Comment:
What is the utility of this function ? we just call an other function... Maybe we can add the `auto_file` instead of `value` ?

Reply:
You are right I will change it to unit type